### ISSUE TITLE
Fix Zoom

### ISSUE THREAD

**Issue by klvs** [klvs/fbspy#16]
Zooming is like 90% done. There's a bit of a bug with it wherein the axis don't scale.

**Comment by klvs** [comment]
@bodeyp1234 @idokin How do you guys feel about abandoning this one if it takes to long? We've spent a ton of time on it and haven't made much headway? At the same time, we do really need interactivity...

Thoughts?

**Comment by bodeyprovansal** [comment]
Yeah, we might have to. I'll keep on it, but I don't have much experience with javascript so I'll take the fall for the lack of progress. @klvs @idokin Is there another task I can work on aside from this that you guys haven't started yet?

**Comment by idokin** [comment]
@bodeyp1234 you can start working on the control panel, I know @klvs has been planning to do it, maybe he can tell you what he had in mind for it.
